medusa/indexers/tvdbv2/tvdbv2_api.py

```diff
@@ -340,7 +341,8 @@ def _parse_images(self, sid):
                 if image_type not in _images:
                     _images[image_type] = {}
 
-                images = self.series_api.series_id_images_query_get(sid, key_type=image_type)
+                images = self.series_api.series_id_images_query_get(sid, key_type=image_type,
+                                                                    accept_language=self.config['language'])
```

Comment:
This is the meat of the PR. I added an Italian show and got only some images, with the language added I get the expected images (all images available on thetvdb).